### ISSUE TITLE
changing profiles down deletes better quality movie

### ISSUE THREAD

**Issue by juggler2** [Radarr/Radarr#1863]
I recently set the profiles down to WEBDL(quality) and then today realised that Radarr has deleted anything with a higher quality than that specified in the profile. Even though these movies are not set to be monitored. Is this desired functionality? I don't understand why we'd like this as a feature. Rather would it be better to have Radarr just then ignore or even recognise that you have a better quality download and leave it?

**Comment by Firemad03** [comment]
+1 I had this bug too.

Radarr should ignore it if better quality and add an advance option to delete them if we want to.

Someone can use this feature to save some disk space.

**Comment by hotio** [comment]
Where are the logs from when radarr deleted the files?

**Comment by juggler2** [comment]
Here are logs from the time the files are showing in the app as being deleted. Is this enough?

![capture](https://user-images.githubusercontent.com/30691517/29118124-a63fa200-7cf8-11e7-8327-dbb457ca4790.JPG)

> 17-8-2 11:12:31.6|Info|Bootstrap|Starting Radarr - /opt/Radarr/Radarr.exe - Version 0.2.0.778
> 17-8-2 11:12:45.3|Info|MigrationLogger|*** Migrating data source=/home/radarr-user/.config/Radarr/nzbdrone.db;cache size=-10485760;datetimekind=Utc;journal mode=Wal;pooling=True;version=3 ***
> 17-8-2 11:12:46.6|Info|MigrationLogger|*** Migrating data source=/home/radarr-user/.config/Radarr/logs.db;cache size=-10485760;datetimekind=Utc;journal mode=Wal;pooling=True;version=3 ***
> 17-8-2 11:12:47.1|Info|Router|Application mode: Interactive
> 17-8-2 11:12:47.1|Info|OwinHostController|Listening on the following URLs:
> 17-8-2 11:12:47.1|Info|OwinHostController|  http://*:7878/radarr/
> 17-8-2 11:12:47.1|Info|OwinHostController|  http://*:7878/
> 17-8-2 11:12:48.4|Info|NancyBootstrapper|Starting NzbDrone API
> 17-8-2 11:13:33.6|Info|RefreshMovieService|Skipping refresh of movie: 2010
> 17-8-2 11:13:33.8|Info|RenameMovieFileService|Renaming movie folder for 2010
> 17-8-2 11:13:33.9|Info|DiskScanService|Scanning disk for 2010
> 17-8-2 11:13:34.4|Info|DiskScanService|Completed scanning disk for 2010
> 17-8-2 11:13:34.5|Info|RefreshMovieService|Skipping refresh of movie: Alien
> 17-8-2 11:13:34.5|Info|DiskScanService|Scanning disk for Alien
> 17-8-2 11:13:34.5|Info|RenameMovieFileService|Renaming movie folder for Alien
> 17-8-2 11:13:34.7|Info|DiskScanService|Completed scanning disk for Alien

**Comment by hotio** [comment]
No this does not give valueble info

**Comment by juggler2** [comment]
thanks please suggest what i should be looking for

**Comment by hotio** [comment]
A debug log which contains the info on why radarr decided to delete the file...

**Comment by hotio** [comment]
Are you even sure it was radarr?

**Comment by juggler2** [comment]
i don't have debug logs i am realising. this wasnt set. without debug logs i cannot say for sure i must be honest. another user seems to have noticed this same occurrence.

radarr has logged when the files were deleted (image in earlier post). this is my primary reason for focusing on radarr. whether this means that radarr had a look to see if the files were still there, saw that _something else_ had deleted them and logged it or whether radarr actively removed the files and logged that, i don't know enough about radarr to tell. i've turned on debug logging now.

given i can't provide more detail if you wish to close this down without further investigating i understand.

**Comment by hotio** [comment]
I'm pretty sure if I manually delete the file and do a rescan it will produce the same screenshot and iirc radarr only tries to delete a file when it can upgrade to a better quality.

**Comment by juggler2** [comment]
You are exactly right. I've just manually renamed a file extension and rescanned. radarr recorded the file as being deleted but did not delete it itself. so this is indeed not an indicator of any issue.

When I have time I can hardlink all my movies to another folder and test changing the profiles to lower qualities with debugging on. that should be fairly conclusive one way or the other?

**Comment by galli-leo** [comment]
@juggler2 I don't think that will be necessary. If Radarr deletes a file, it should have a log entry, even with no debug Log. (something like: Recycle bin not configured, deleting permanently). Thus I am pretty sure it cannot have been Radarr. Did you have any other programs running, which were managing your movie library? (Plex, Download Client, etc.)

**Comment by juggler2** [comment]
Thank's @galli-leo I've realised this wasn't radarr now. I used 'cp -al' command to create a hardlink of the directory tree and then changed profiles and nothing happened. So i was definitely wrong here.

I did have rsync running which is the only thing I can think of. That is a cron job but it's not been changed since and still seems ok. Who knows but yes, definitely not radarr.

thanks for coming back to me!
